test/e2e/plain_provisioner_test.go

```diff
@@ -279,40 +246,14 @@ var _ = Describe("plain provisioner bundle", func() {
 		})
 
 		It("checks the bundle's phase is stuck in pending", func() {
-			By("waiting until the pod is reporting ImagePullBackOff state")
-			Eventually(func() bool {
-				pod := &corev1.Pod{}
-				if err := c.Get(ctx, types.NamespacedName{
-					Name:      bundleDeployment.GetName(),
-					Namespace: defaultSystemNamespace,
-				}, pod); err != nil {
-					return false
-				}
-				if pod.Status.Phase != corev1.PodPending {
-					return false
-				}
-				for _, status := range pod.Status.ContainerStatuses {
-					if status.State.Waiting != nil && status.State.Waiting.Reason == "ImagePullBackOff" {
-						return true
-					}
-				}
-				return false
-			}).Should(BeTrue())
-
 			By("waiting for the bundle to report back that state")
 			Eventually(func() bool {
 				err := c.Get(ctx, client.ObjectKeyFromObject(bundleDeployment), bundleDeployment)
 				if err != nil {
 					return false
 				}
 				unpackPending := meta.FindStatusCondition(bundleDeployment.Status.Conditions, rukpakv1alpha2.PhaseUnpacked)
-				if unpackPending == nil {
-					return false
-				}
-				if unpackPending.Message != fmt.Sprintf(`Back-off pulling image "%s"`, bundleDeployment.Spec.Source.Image.Ref) {
-					return false
-				}
-				return true
+				return unpackPending != nil
```

Comment:
This test no longer seems to be testing what it says it is testing.  Before, it only finished successfully when the unpack pod had "Back-off pulling image".

Now, it finishes successfully if it finds an unpacked condition with any status/message?

Reply:
Ah good catch - I went through and removed all the pod based tests. Seems like I missed that this was testing for a specific unpack state. I'll fix this

Reply:
Fixed in https://github.com/operator-framework/rukpak/pull/874/commits/9bc11577670c2a52254ae026c64f0336f6e32117